tecton_client/_internal/request_utils.py

```diff
@@ -0,0 +1,68 @@
+from typing import Dict
+
+
+class MetadataOptions:
+    """Passed into metadata_options on get_features, controls what metadata is returned as part of the response."""
+
+    def __init__(
+        self,
+        include_names: bool = True,
+        include_data_types: bool = True,
```

Comment:
Feels funky that some default to True and some default to false right? Because now `MetadataOptions()` is not actually empty.

Reply:
Mildly, but I think this is correct behavior. `True` is definitely the right default value for `include_names` and `include_data_types`. `get_features_dict()` just straight up doesn't work when either of those are False. 

If this (without `metadata_options` supplied):
```
get_features(feature_service_name='fv1', join_key_map={'user_id': '123'})
```
defaults names/data_types to True and `get_features_dict` works, but then the user ties to also request `include_serving_status`:
```
get_features(feature_service_name='fv1', join_key_map={'user_id': '123'}, 
    metadata_options=MetadataOptions(include_serving_status=True)
```
and it changes the behavior of `include_names`/`include_data_types`/`get_features_dict`, that would be surprising to them. 

I think the mental model is "These are options but they have different default values". You could see these could have non-boolean options in the future as well.